site/content/entry/citra-progress-report-2017-october-november.md

```diff
@@ -0,0 +1,101 @@
++++
+date = "2018-01-10T13:11:00-04:00"
+title = "Citra Progress Report - 2017 October and November"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 0
++++
+
+Past the red leaves and into the icy tundra in the midst of winter comes the present.
+This time we'll be covering not one, but two months of progress! With a great many
+changes all around, this article is going to be packed to the brim with all the
+new goodies that have come to Citra those crimson months.
```

Comment:
> come to Citra those crimson months

come to Citra _during_ those crimson months

Not sure what crimson means in this context.

Reply:
Autumn in temperate zones usually looks like this: 
![](https://user-images.githubusercontent.com/10248424/35127217-77deaef6-fc87-11e7-97ba-64cabf701a54.jpg)

Very red, as you might tell. :stuck_out_tongue:

Reply:
Oh now I get it 😅